templates/templates/preview/model.template

```diff
@@ -0,0 +1,87 @@
+{%- if model is defined -%}Model name: {{ model.name }}
+        Executable:  
+        {%- for param in model.run_settings.exe %}
+            {{ param }}
+        {%- endfor %}
+        Executable Arguments:
+        {%- for param in model.run_settings.exe_args %}
+            {{ param }}
+        {%- endfor %}
+        {%- if model.batch_settings -%}
+        Batch Launch: True
+        {% else %}
+        Batch Launch: False
+        {%- endif -%} 
+        {% if model.params -%}
+        Model parameters: 
+        {%- for param, value in model.params.items() %}
+            {{ param }}: {{ value }}
+        {%- endfor -%}
+        {% endif %}
+        {%- if model.files %}
+        {%- if model.files.tagged %}
+        Tagged Files for model configuration: 
+            {{ model.files.tagged[0] }} -> 
+            {{ model.path }}
+        {%- endif -%}
+        {%- if model.files.copy %}
+        Copy files: 
+            {{ model.files.copy[0] }} ->
```

Comment:
is hardcoding the 0-th element ok?

Reply:
good point. Changed to for loop. Will have to double check how multiple files should be displayed